### PR TITLE
Fix screen capture to work on windows

### PR DESCRIPTION
**Pull request by rchiodo**
Screen capture file names are too long. I can't open them on windows. 

This should achieve the same result (unique file names)

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -223,7 +223,7 @@
             ],
             "env": {
                 "VSC_JUPYTER_FORCE_LOGGING": "1",
-                "VSC_JUPYTER_CI_TEST_GREP": "", // Leave as `VSCode Notebook` to run only Notebook tests.
+                "VSC_JUPYTER_CI_TEST_GREP": "Ensure ipykernel install prompt is displayed every time you try to run a cell in an Interactive Window and you can switch kernels", // Leave as `VSCode Notebook` to run only Notebook tests.
```
Comment (rchiodo) [inline .vscode/launch.json:226]: Whoops

File: src/test/common.node.ts [hunk 2]
```diff
@@ -322,21 +323,15 @@ export async function captureScreenShot(contextOrFileName: string | Mocha.Contex
...
-            : hashjs
-                  .sha256()
-                  .update(contextOrFileName.currentTest?.fullTitle() || '')
-                  .digest('hex');
     const testTitle = typeof contextOrFileName === 'string' ? '' : contextOrFileName.currentTest?.title || '';
     const fileNamePrefix =
-        typeof contextOrFileName === 'string' ? contextOrFileName : `${testTitle}_${fullTestNameHash}`;
-    const name = `${fileNamePrefix}_${uuid()}`.replace(/[\W]+/g, '_');
+        typeof contextOrFileName === 'string' ? contextOrFileName : `${testTitle}_${screenShotIndex}`;
+    const name = fileNamePrefix.replace(/[\W]+/g, '_');
```
Comment (DonJayamanne) [inline src/test/common.node.ts:329]: Can we keep the hash and the screenshot index and just remove the `uuid`.
I think that will reduce the length.
Optionally we could also reduce the length of the test title by taking just the first 100 characters `testTitle.substring(0, 100)`

The hash is used in another place, so we can determine the screenshots that belong to a particular test.
Basiscally we now generate ipynb files with the test failures and screenshots of the errors, i demnoe'd this last week, probably should share this again.

If  you go into the CI job and look for artifacts named 'TestResultNotebook...ipynb` that will contain the failed tests, with test logs for that test and the corresponding screenshots as well.
I find that very useful, else I have to parse the entire log file manually & then look for screenshots manually, this way the ipynb has everything in one place.

**Review by DonJayamanne — CHANGES_REQUESTED** [review]
Please leave the hash, and you could just remove the uuid and keep the screenshot index.

File: src/test/common.node.ts (hunk 2 above)
Comment (rchiodo) [reply src/test/common.node.ts:329]: I wonder if that notebook works on windows at the moment. I should try that and see.

Comment (rchiodo) [reply src/test/common.node.ts:329]: Hmm, I don't see a failure with screenshots.

Comment (rchiodo) [reply src/test/common.node.ts:329]: Did you remove the separate screenshots now that we have a notebook? 

On this run, I would have expected there to be screenshots:
https://github.com/microsoft/vscode-jupyter/actions/runs/2659693681

**Comment by rchiodo** [comment]
Looks like screenshots aren't working at all before this, so closing for now. I'll retry opening screenshots after we get screenshots working again.

**Comment by DonJayamanne** [comment]
I don't think this is required anymore.

**Pull request closed without merging**